doc/modules/ROOT/pages/quickstart.adoc

```diff
@@ -0,0 +1,178 @@
+
+* xref:cloud:getting-started.adoc[Hazelcast Cloud]
+* xref:hazelcast:getting-started:quickstart.adoc[Hazelcast CLI]
+* xref:hazelcast:getting-started:get-started-binary.adoc[Binary distributions]
+
+If you want to use Hazelcast Cloud, follow xref:cloud:net-client.adoc[these instructions].
+
+Alternatively, the Hazelcast .NET client https://github.com/hazelcast/hazelcast-csharp-client[repository] on GitHub provides a
+https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.2[PowerShell script] which can be used to download and run a Hazelcast test server. For instance, the following downloads and starts a server version {server-full-version}:
+
+[source,shell,subs="attributes+"]
+----
+PS> ./hz.ps1 run-server -server {server-full-version}
+----
+====
+
+== Step 3. Connect to the Cluster
+
+Once the client is installed and a default cluster is started, you're ready to use the Hazelcast client in your .NET application.
+Connect your client to the cluster with this single line:
+
+[source,csharp]
+----
+await using var client = await HazelcastClientFactory.StartNewClientAsync();
+----
+
+This is enough to connect to the cluster you've created in Step 2 with default settings. When you run it, you will see the client is connected to the cluster:
+
+[source,shell,subs="attributes+"]
+----
+info: Hazelcast.Clustering.Heartbeat[0]
+      Heartbeat with 5000ms period and 60000ms timeout
+info: Hazelcast.Clustering.ClusterConnections[0]
+      Authenticated client 'hz.client_3' (3121b61) running version 5.0.0+d5ffe4 on connection 9d158e6 from 127.0.0.1:54260 to member c347caf at 127.0.0.1:5701 of cluster 'dev' (9d4d52e) running version {server-full-version}.
+info: Hazelcast.Clustering.ClusterMembers[0]
+      Members [1] {
+          Id=c347caf Internal=10.5.0.2:5701* Public=<none> Adding
+      }
+info: Hazelcast.Clustering.ClusterConnections[0]
+      Authenticated client 'hz.client_3' (3121b61) running version 5.0.0+d5ffe4 on connection 7b73a18 from 10.5.0.2:54261 to member c347caf at 10.5.0.2:5701 of cluster 'dev' (9d4d52e) running version {server-full-version}.
+----
+
+You can also build the `HazelcastOptions` factory with options and start the client to connect to the cluster with the provided options.
+A simple example is as follows.
+
+[source,csharp]
+----
+var options = HazelcastOptions.Build();
+options.Networking.Addresses.Add("127.0.0.1"); <1>
+options.ClusterName = "dev"; <2>
+options.ClientName = "MyClient"; <3>
+
+await using var client = await HazelcastClientFactory.StartNewClientAsync(options); <4>
+----
+<1> IP addresses of one or more servers in the cluster. For this example, this is the localhost (127.0.0.1) from Step 2 since you've created a single-server cluster on your local machine.
+If you have multiple servers, you can provide their addresses separated by comma. The client uses the addresses provided here to find and connect
```

Comment:
.Net client doesn't support multiple address in a single `Add` seperated by coma but port can be provided with colon like `options.Networking.Addresses.Add("127.0.0.1:5701");` To add another address, the Add method should be called again, like `options.Networking.Addresses.Add("127.0.0.1:5702");`